setup.py

```diff
@@ -33,6 +34,7 @@
     'reactivex',
     'retry',
     'scipy',
+    'sentence_transformers',
```

Comment:
Nit: no longer used but I think your new embedding changes will be useful. Shall we do that as a seperate PR?

Reply:
By adding this here, it is just a command line later in the examples. can we keep as part of this commit?

Reply:
If it's for the examples it shoudl go in `examples/requirements.txt` please. Not adding it if it's not used by our source code means that people don't need this dep to install `concordia`. So if this dep everdoesn't support our python version or M1 macs then it doesn't block others.

Reply:
done